src/e2e/java/teammates/e2e/pageobjects/AdminNotificationsPage.java

```diff
@@ -0,0 +1,232 @@
+package teammates.e2e.pageobjects;
+
+import static org.junit.Assert.assertEquals;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import teammates.common.datatransfer.NotificationStyle;
+import teammates.common.datatransfer.NotificationTargetUser;
+import teammates.common.datatransfer.attributes.NotificationAttributes;
+
+/**
+ * Page Object Model for the admin notifications page.
+ */
+public class AdminNotificationsPage extends AppPage {
+
+    @FindBy(id = "btn-add-notification")
+    private WebElement addNotificationButton;
+
+    @FindBy(id = "notifications-timezone")
+    private WebElement notificationsTimezone;
+
+    @FindBy(id = "target-user")
```

Comment:
Would you be able to refactor `target-user` and `message`, since most of the attributes are already prefixed with `notifcation-xx`